coast/data/gridded.py

```diff
@@ -94,23 +94,19 @@ def _setup_grid_obj(self, chunks, multiple, **kwargs):
             )  # THIS ADDS TO dataset_domain. Should it be 'return'ed (as in trim_domain_size) or is implicit OK?
             self.merge_domain_into_dataset(dataset_domain)
             debug(f"Initialised {get_slug(self)}")
-        ###################
-        Make_LonLat_2D = kwargs.get("Make_LonLat_2D", False)
-        if Make_LonLat_2D:
```

Comment:
think we still need to check for this key word and call the new function

Reply:
I was planning on updating the tests to call the new function as part of the workflow, rather than relying on passing kwargs as flags to the __setpu_grid_obj method....